apps/src/templates/sectionAssessments/MultipleChoiceAnswerCell.jsx

```diff
@@ -24,10 +24,26 @@ class MultipleChoiceAnswerCell extends Component {
   static propTypes = {
     percentValue: PropTypes.number.isRequired,
     isCorrectAnswer: PropTypes.bool,
+    answerCell: PropTypes.string,
```

Comment:
these are all answer cells, could we call it something like `singleStudentResponse`?